adform-htb.js

```diff
@@ -249,55 +350,35 @@ function AdformHtb(configs) {
             headerStatsInfo[htSlotId] = {};
             headerStatsInfo[htSlotId][curReturnParcel.requestId] = [curReturnParcel.xSlotName];
 
-            var curBid;
-
-            for (var i = 0; i < bids.length; i++) {
-
-                /**
-                 * This section maps internal returnParcels and demand returned from the bid request.
-                 * In order to match them correctly, they must be matched via some criteria. This
-                 * is usually some sort of placements or inventory codes. Please replace the someCriteria
-                 * key to a key that represents the placement in the configuration and in the bid responses.
-                 */
-
-                /* ----------- Fill this out to find a matching bid for the current parcel ------------- */
-                if (curReturnParcel.xSlotRef.someCriteria === bids[i].someCriteria) {
-                    curBid = bids[i];
-                    bids.splice(i, 1);
-                    break;
-                }
-            }
-
-            /* No matching bid found so its a pass */
-            if (!curBid) {
-                if (__profile.enabledAnalytics.requestTime) {
```

Comment:
Please add this check back in. This check validates if the current bid exists.

Reply:
There is no need to check this, there always would be the same number of response items as requested.

Reply:
This is more for the case where the bid response didn't come back or invalid. 
The `__baseClass._emitStatsEvent(sessionId, 'hs_slot_pass', headerStatsInfo);` will also emit stats event for cases like this.